studio/.gitignore

```diff
@@ -33,3 +33,4 @@ yarn-error.log*
 .env.production.local
 
 !.vercel
+.vercel
```

Comment:
i think this is conflicting with line 35?  do we want to ignore `.vercel`? or dont ignore `.vercel`?